src/components/Comments.jsx

```diff
@@ -19,13 +23,66 @@ const Comments = ({ article_id, comments, setComments }) => {
       });
   }, [article_id, setComments]);
 
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    const newComment = {
+      article_id: article_id,
+      username: username,
+      body: commentBody,
+    };
+    postComment(newComment)
+      .then((response) => {
+        setHasPosted(true);
+        setComments([response.data.comment, ...comments]);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(err);
```

Comment:
This error state is also shared with the loading of comments. We should have separate messages to the user. If a comment posting fails, I may want to try again rather than having to refresh the page and lose what I've entered into the text field.